src/MergeContext.js

```diff
@@ -525,23 +527,23 @@ class FieldsTokenizer
                 break;
 
             if (/^\s/.test(line))
-                throw new Error(`a field cannot start with whitespace: ${line}`);
+                throw new Error(`Invalid ${tag}: a field cannot start with whitespace: '${line}'`);
```

Comment:
These tags should provide some additional information to generic problem sources.

Reply:
Please call them parsingContext (instead of more generic "tag") to ease code search and maintenance.